dev_guide/builds.adoc

```diff
@@ -1426,6 +1432,96 @@ Configuration change triggers currently only work when creating a new
 able to launch a build whenever a `*BuildConfig*` is updated.
 ====
 
+[[build-hooks]]
+
+== Build Hooks
+
+Build hooks can be used as a part of the build flow to run tests inside a
+temporary container and validate your image before it is pushed to a
+registry. The command is executed with the current working directory set to the
+image's `*WORKDIR*` which is your current working directory where your code is
+located. The hook will fail if the script or command returns a non-zero exit
+code or if there's any other error related to starting the temporary container.
+When the hook fails it will mark the build as failed and the image will not be pushed to a registry.
+====
+[source,yaml]
+spec:
+  postCommit:
+    script: "rake test" <1>
+
+<1> `script` is a shell script to be run with `/bin/sh -c`. Use this when a
```

Comment:
Not 100% sure, but thinking ``` `script` ``` is markdown syntax, and we normally use ``` `*script*` ``` in our adocs...